kolibri/plugins/style_guide/assets/src/views/content/Checkboxes/index.vue

```diff
@@ -44,17 +46,18 @@
 
 <script>
 
-  import FullVue from 'vue/dist/vue.common';
   import KCheckbox from 'kolibri.coreVue.components.KCheckbox';
   import ComponentDocs from '../../shell/ComponentDocs';
   import VueExample from '../../shell/VueExample';
   import PageTemplate from '../../shell/PageTemplate';
   import Show from '../../shell/Show';
 
-  import example from 'raw-loader!./example.html';
-  import kCheckboxApi from '!vue-doc!kolibri.coreVue.components.KCheckbox';
+  /* eslint-disable import/no-duplicates */
+  import Example from './Example.vue';
```

Comment:
Remove the "Checkboxes" h1 tag at the top, since it's already in the fixed header.

Reply:
Same with "Horizontal navbar", "topic tree breadcrumbs" and "Tooltips" pages